apps/vmq_server/priv/vmq_server.schema

```diff
@@ -269,6 +269,21 @@
                                                                   {datatype, atom},
                                                                   hidden]}.
 
+%% @doc specify how offline messages should be delivered if a session switches from offline
+%% to online. Currently, two strategies are supported: 'adhere_queue_size' or 'add_all'.
+{mapping, "queue_offline_online_strategy", "vmq_server.queue_offline_online_strategy", [{default, adhere_queue_size},
+                                                  {datatype, atom},
```

Comment:
datatype can be a flag instead of enum, then you can just set a single setting to on or off. (`override_max_online` or sth similar)

Reply:
Could do that. The idea was being able to add more sophisticated "strategies", if needed.

Reply:
I think the current approach takes us a long way before more strategies are needed. I also think a flag just makes it easier for users, so I prefer that.

Reply:
Can you change the datatype to enum instead of atom?